packages/wonder-blocks-timing/src/util/types.flowtest.ts

```diff
@@ -19,8 +19,8 @@ type Props1 = {
     test: string;
 } & WithActionSchedulerProps;
 
-// @ts-expect-error [FEI-5019] - TS2322 - Type 'string' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'.
 const InnerComponent1: React.FC<Props1> = (props): React.ReactElement =>
+    // @ts-expect-error [FEI-5019] - TS2322 - Type 'string' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'.
     props.test;
```

Comment:
This can be updated to use the `<>{string}</>` so we can drop this `ts-expect-error`.